app/views/application/suitability/index.njk

```diff
@@ -45,17 +48,17 @@
 
 {{ govukRadios({
   fieldset: {
-    classes: "govuk-!-margin-top-6",
+    classes: "govuk-!-margin-top-7",
     legend: {
-      text: "Do you want to share any information which could have an impact on your application?",
+      text: "Do you want to share any safeguarding issues?",
       classes: "govuk-fieldset__legend--m"
     }
   },
   items: [{
     value: "true",
     text: "Yes, I want to share something",
     hint: {
-      text: "You can tell us about it here or get in touch with your training provider directly."
+      text: "You talk to your provider directly or tell us about it here. This won’t stop you from submitting your application."
```

Comment:
Do we need the "This won’t stop you from submitting your application" bit?

Reply:
This was because there were concerns from candidates about whether they'd still be able to continue with their application